bigchaindb/models.py

```diff
@@ -347,7 +348,8 @@ def from_db(cls, bigchain, block_dict):
         asset_ids = cls.get_asset_ids(block_dict)
         assets = bigchain.get_assets(asset_ids)
         block_dict = cls.couple_assets(block_dict, assets)
-        return cls.from_dict(block_dict)
+        kwargs = from_dict_kwargs or {}
```

Comment:
Can't we just remove this line and do `from_dict_kwargs={}` in the method parameters?